sys/sys/mbuf.h

```diff
@@ -419,7 +419,8 @@ struct mbuf {
 				char		m_pktdat[0];
```

Comment:
So why does this not need `__subobject_use_remaining_size`?

Reply:
Should be implicit for any variable-length fields ([0] is treated as variable)

Reply:
See above